tests/my_test_Inference_RAxML.py

```diff
@@ -0,0 +1,264 @@
+            boolvals = [name in clade_term_names for name in term_names]
+            bitstr = _BitString("".join(map(str, map(int, boolvals))))
+            bitstrs.add(bitstr)
+        return bitstrs
+
+    def compare(self, tree1, tree2):
+        term_names1 = [term.name for term in tree1.get_terminals()]
+        term_names2 = [term.name for term in tree2.get_terminals()]
+        # false if terminals or BitStrings are not the same
+        return set(term_names1) == set(term_names2) and self._bitstrs(tree1) == self._bitstrs(tree2)
+
+    @pytest.mark.parametrize("format_list", [(["fasta", "phylip"])])
+    def test_sprt_infile_formats(self, format_list: list):
+        """
+        Test function to check that RAxML module supports the required infile formats.
+        Arguments :
+            format_list: List of accepted infile formats for using RAxML tool.
+        """
+        assert len(Rax.SPRT_INFILE_FORMATS) == len(format_list)
+        assert Rax.SPRT_INFILE_FORMATS == format_list
+
+    @pytest.mark.parametrize(
+        "keywords",
+        [
+            (
+                {
+                    "default": ["-m", "GTRCAT", "--silent"],
+                    "JC+CAT": ["-m", "GTRCAT", "--JC69", "--silent"],
+                    "JC+G": ["-m", "GTRGAMMA", "--JC69", "--silent"],
+                    "K80+CAT": ["-m", "GTRCAT", "--K80", "--silent"],
+                    "K80+G": ["-m", "GTRGAMMA", "--K80", "--silent"],
+                    "HKY+CAT": ["-m", "GTRCAT", "--HKY85", "--silent"],
+                    "HKY+G": ["-m", "GTRGAMMA", "--HKY85", "--silent"],
+                    "GTR+CAT": ["-m", "GTRCAT", "--silent"],
+                    "GTR+G": ["-m", "GTRGAMMA", "--silent"],
+                    "JTT+CAT": ["-m", "PROTCATJTT", "--silent"],
+                    "JTT+G": ["-m", "PROTGAMMAJTT", "--silent"],
+                    "WAG+CAT": ["-m", "PROTCATWAG", "--silent"],
+                    "WAG+G": ["-m", "PROTGAMMAWAG", "--silent"],
+                }
+            ),
+        ],
+    )
+    def test_keywords(self, keywords: dict):
+        """
+        Test function to check that RAxML module has the same keys and values for the keywords dictionary,
+        that will be used later to pass arguments to a command.
+        Arguments :
+            keywords: Dictionary where keys are shortcuts for arguments combinations and values are lists of
+            arguments a command may receive.
+        """
+        assert len(Rax.KEYWORDS) == len(keywords)
+        for aKey, bKey in zip(Rax.KEYWORDS.keys(), keywords.keys()):
+            assert aKey == bKey
+        for key in Rax.KEYWORDS:
+            assert Rax.KEYWORDS[key] == keywords[key]
+
+    @pytest.mark.parametrize(
+        "args, infile_path, bootstraps, tmpdir_path, seed, arg_list",
+        [
+            (
+                "JC+CAT",
+                "tests/Fasta/f001.mafft_linsi.aln",
+                1,
+                tmp_dir,
```

Comment:
I struggle to understand the need for this attribute and `tmpdir_path`. Why not just use `tmp_path` fixture of `pytest`?

Reply:
You're right, my bad.